client/reader/controller.js

```diff
@@ -44,7 +44,7 @@ function renderFeedError( context ) {
 	);
 }
 
-module.exports = {
+export default {
```

Comment:
👍 I've gone ahead and codemodded all instances of default export objects to use named exports.